std/json.d

```diff
@@ -1025,11 +1027,10 @@ string toJSON(in JSONValue* root, in bool pretty = false, in JSONOptions options
                 case '\r':      json.put("\\r");        break;
                 case '\t':      json.put("\\t");        break;
                 default:
-                    appendJSONChar(&json, c,
-                                   (msg) { throw new JSONException(msg); });
+                    appendJSONChar(&json, c, options,
+                        (msg) { throw new JSONException(msg); });
```

Comment:
Nitpick: lining this line up with the opening `(` (like it was originally) looks nicer. Not sure why the indent was changed when all that was needed was to append `options` to the end of the previous line.